keanu-project/src/main/java/io/improbable/keanu/tensor/dbl/Nd4jDoubleTensor.java

```diff
@@ -1029,7 +1029,7 @@ public BooleanTensor elementwiseEquals(Tensor that) {
         }
     }
 
-    static INDArray unsafeGetNd4J(DoubleTensor that) {
+    static public INDArray unsafeGetNd4J(DoubleTensor that) {
```

Comment:
Does this need to be public? This should be implementation specific and therefore private to this class.